client/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,104 @@
+import {
+  AppBar, Container, Link,
+} from '@mui/material';
+import {
+  AiFillFacebook, AiFillTwitterCircle, AiFillInstagram, AiOutlineWhatsApp,
+} from 'react-icons/ai';
+import HomeIcon from '../../assets/homeIcon.png';
+
+import './footer.css';
+
+export default function Footer() {
+  return (
+
+    <AppBar
+      position="fixed"
+      color="primary"
+      sx={{ top: 'auto', bottom: 0 }}
+      style={{
+        background: '#3381CB', height: '432px', alignItems: 'center',
+      }}
+    >
+      <Container
+        maxWidth="xl"
+        sx={{
+          display: 'flex',
+          justifyContent: 'space-around',
+          marginTop: '10px',
+          height: '100%',
+          alignItems: 'center',
+        }}
+      >
+        <section className="img">
+          <img
+            src={HomeIcon}
+            alt="home icon"
+            loading="lazy"
+            style={{ borderBottomLeftRadius: 4, borderBottomRightRadius: 4, width: '6rem' }}
+          />
+
+        </section>
+        <section className="section2">
```

Comment:
Whenever you see code being repetitive, then you're doing something wrong.
All these sections and links could be generalized.
You could have an object like this:
```
[
            {
              title: 'About us',
              links: [
                {href: 'http://', name: 'Story', /** more props */}, 
                {href: 'http://', name: 'Clients', /** more props */}, 
                ...
              ]
            }
          ]
```
And spread your properties accordingly.
Advantage of using this style, is you can toggle on/off any more properties you wish to add.